src/hostapi/dsound/pa_win_ds.c

```diff
@@ -3051,9 +3051,7 @@ static PaError StartStream( PaStream *s )
 #ifndef PA_WIN_DS_USE_WMME_TIMER
     if( stream->processingThread )
     {
-#ifdef CLOSE_THREAD_HANDLE
```

Comment:
The purpose of `#ifdef CLOSE_THREAD_HANDLE` is to allow the macro to be a no-op. Given that there is now no situation where it would be a no-op I am not sure that the macro should be present at all.